config/locales.json

```diff
@@ -76,5 +76,17 @@
     "locale": "Vietnamese",
     "Đ": "D",
     "đ": "d"
+  },
+  "da": {
+    "locale": "Danish",
+    "Æ": "AE",
+    "Ø": "OE",
+    "Å": "AA",
```

Comment:
Can you add the lower case letters as well? Also `Æ` seems to be part of the main charmap with correct transliteration in your case.

Reply:
Yes, thank you.